torchdata/dataloader2/adapter.py

```diff
@@ -53,9 +53,21 @@ class Shuffle(Adapter):
             - None: No-op. Introduced for backward compatibility.
 
     Example:
-        >>>  dp = IterableWrapper(range(size)).shuffle()
-        >>>  dl = DataLoader2(dp, [Shuffle(False)])
-        >>>  self.assertEqual(list(range(size)), list(dl))
+
+    .. testsetup::
+
+        from torchdata.datapipes.iter import IterableWrapper
+        from torchdata.dataloader2 import DataLoader2
+        from torchdata.dataloader2.adapter import Shuffle
+
+        size = 12
+
+    .. testcode::
+
+        dp = IterableWrapper(range(size)).shuffle()
+        dl = DataLoader2(dp, [Shuffle(False)])
+        assert list(range(size)) == list(dl)
+
```

Comment:
Could you please provide a screen shot of this change on our online doc?
It was like https://pytorch.org/data/beta/generated/torchdata.dataloader2.adapter.Shuffle.html#torchdata.dataloader2.adapter.Shuffle

Reply:
This is how to build doc: https://github.com/pytorch/data/tree/main/docs#building-the-documentation

Reply:
And here is the way to preview the change locally https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md#previewing-changes-locally

Reply:
Here is a side by side comparison of the docs before (left) and after the changes (right). Nothing has changed except that it is now executable with doctest as there is some setup code (sphinx hides it from the docs) for the imports and the size parameter.

<img width="1720" alt="shuffle_docs" src="https://user-images.githubusercontent.com/15790457/197520545-4a675a80-72f9-4587-bf50-f688faa63642.png">

Reply:
That's great! Thank you!